### ISSUE TITLE
reCaptcha not working on User Registration Page

### ISSUE THREAD

**Issue by kwalski** [CakeDC/users#351]
I can't see how to configure reCaptcha API Keys in the plugin's docs.
So, I tried configuring it in app/config/bootstrap.php and it didn't work.

On the registration page I am getting error `reCaptcha is not configured! Please configure reCaptcha.key or set Users.Registration.reCaptcha to false`

May I suggest update the UserHelper docs with an instruction on where to configure api key for reCaptcha?

**Comment by steinkel** [comment]
Yes, that's not clear in the docs now, we'll improve it. Thanks!

**Comment by coolex** [comment]
I put in bootstrap:
Plugin::load('CakeDC/Users', ['routes' => true, 'bootstrap' => true]);
Configure::write('reCaptcha.key','************');
and it's work.

But registration page not check if is reCaptcha valid.

What I miss?

**Comment by steinkel** [comment]
@coolex we are planning to improve this a bit giving the ability to configure where the recaptcha should be used (registration/login). Thanks

**Comment by kwalski** [comment]
For the time being I could see reCaptcha using  Configure::write('reCaptcha.key', <Key>)' in the bootstrap taking hint from @coolex 
I have put another line for reCaptcha.secret key
However on submit, I am getting error 
```
Error: Class 'ReCaptcha\ReCaptcha' not found 
File vendor/cakedc/users/src/Controller/Traits/ReCaptchaTrait.php
Line: 52
```
Is ReCaptcha in working order?

**Comment by steinkel** [comment]
@kwalski please ensure you do `composer require google/recaptcha:@stable`

We'll update the docs soon and improve recaptcha features a bit...

Thanks,

**Comment by steinkel** [comment]
closed by #355
